pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -2731,7 +2731,7 @@ protected void internalSetMaxUnackedMessagesPerConsumer(int maxUnackedMessagesPe
         }
     }
 
-    protected int internalGetMaxUnackedMessagesPerSubscription() {
+    protected Integer internalGetMaxUnackedMessagesPerSubscription() {
```

Comment:
If we want to introduce some change that will break the namespace level policy behavior, is it need to start a PIP to discuss it and it should be a highlight in the release note. @sijie What do you think

Reply:
It is better to have a PIP for this change. We should highlight it in the release notes.